src/styles/main.scss

```diff
@@ -0,0 +1,148 @@
+@mixin properties {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.body {
+  margin: 0;
+
+  @include properties;
+
+  height: 100vh;
+}
+
+.clock {
+  width: 500px;
+  height: 500px;
+
+  @include properties;
+
+  border: 1px solid black;
+  border-radius: 50%;
+
+  position: relative;
+  background-image: url(image.jpg);
+
+  &__seconds-tablo {
+    width: 410px;
+    height: 410px;
+
+    @include properties;
+    position: absolute;
+
+    border-style: dotted;
+    border-radius: 50%;
+    border-width: 11px;
+  }
+
+  &__hours-box {
+    width: 100%;
+    height: 100%;
+
+    position: absolute;
+  }
+
+  &__minute-hand {
+    z-index: 1;
+    height: 30%;
+    width: 10px;
+
+    background-color: gold;
+    position: absolute;
+    border: 1px solid black;
+    border-radius: 100%;
+    transform-origin: bottom center;
+
+    top: 20%;
+    left: 49%;
+
+    animation: clock__minute-hand 3600s steps(60) infinite;
+  }
+
+  &__second-hand {
+    z-index: 2;
+    height: 40%;
+    width: 6px;
+
+    background-color: blue;
+    position: absolute;
+    border: 1px solid black;
+    border-radius: 100%;
+    transform-origin: bottom center;
+
+    top: 10%;
+    left: 49%;
+
+    animation: clock__second-hand 60s steps(60) infinite;
+  }
+
+  &__dot {
+    z-index: 3;
+    width: 20px;
+    height: 20px;
+    background-color: black;
+    border-radius: 50%;
+    position: absolute;
+  }
+}
+
+.hour-12 {
+  font-size: 60px;
+
+  margin-top: -6px;
+  margin-left: 176px;
+  position: absolute;
+}
+
+.hour-3 {
+  font-size: 60px;
+
+  margin-left: 370px;
+  margin-top: 172px;
+
+  position: absolute;
+}
+
+.hour-6 {
+  font-size: 60px;
+
+  margin-top: 350px;
+  margin-left: 190px;
+  position: absolute;
+}
+
+.hour-9 {
+  font-size: 60px;
+
+  margin-left: 10px;
+  margin-top: 172px;
+
+  position: absolute;
+}
+
+@keyframes clock__second-hand {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes clock__minute-hand {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
they are the same) use one for both cases. And move keyframes to a separate file